appinventor/appengine/src/com/google/appinventor/client/opendyslexic/README.txt

```diff
@@ -0,0 +1,70 @@
+README
```

Comment:
The fonts are duplicated here and in appengine/war/fonts. I don't think that these need to be here and that they should be in appengine/war/fonts.